### ISSUE TITLE
Docs site broken

### ISSUE THREAD

**Issue by martijnrusschen** [Hacker0x01/react-datepicker#2647]
**Describe the bug**
```
react.development.js:1465 Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
```

**To Reproduce**

1. check out the latest master branch
2. run `yarn; yarn build-dev; yarn start`
3. See the error in your console

Is anyone able to troubleshoot what's going wrong? I don't have much time to investigate this currently. I suspect this started to happen after a dependency upgrade. Any help would be appreciated.

**Comment by tu4mo** [comment]
Two different versions of React is imported, likely due to https://github.com/Hacker0x01/react-datepicker/commit/8487dd3ba1e5e14d76a74d003d481edcc9b78342.

**Comment by martijnrusschen** [comment]
@tu4mo You're right, I've put up a fix to revert that change, doesn't look like we to require need react at all there.
